README.md

```diff
@@ -609,18 +609,20 @@ A Trial procedure takes place in the following way:
 2. The prosecutor will outline the actions of the defendant that he claims are in violation of the rules and demand a
 	punishment in the form of a non-negative integer point reduction of the defendants point. This reduction can be of 0
 	points.
-3. The defendant is allowed to defend his actions and can plead either _Guilty_ or _Not Guilty_. If he pleads Guilty, he
-	can propose an alternative punishment in the form of a non-negative integer point reduction no larger than that of the
-	prosecution. If he pleads Not Guilty, an alternative punishment of a zero point reduction is assumed.
+3. The defense attorney is allowed to defend the actions of the defendant and can enter a plea of either _Guilty_ or
+	_Not Guilty_. If a Guilty plea is entered, he can propose an alternative punishment in the form of a non-negative
+	integer point reduction no larger than that of the prosecution. If a Not Guilty plea is entered, an alternative
+	punishment of a zero point reduction is assumed.
```

Comment:
I object to the wording here; if a defended is determined Not Guilty, it would be absurd to incur any punishment on the defendant, even if this punishment consists of a zero point reduction.